manubot/cite/__init__.py

```diff
@@ -113,3 +116,21 @@ def citation_to_citeproc(citation):
     citeproc = citeproc_passthrough(citeproc, set_id=citation_id)
 
     return citeproc
+
```

Comment:
7 lines above this I noticed an extra space in the message `Unsupported citation  source {source} in {citation}`